lib/pwwka/transmitter.rb

```diff
@@ -1,3 +1,5 @@
+require 'resque'
```

Comment:
I'm assuming we don't want to require resque (correct me if I'm wrong), so I'll make this require optional.